packages/core/src/actions/spawnChild.ts

```diff
@@ -103,6 +107,8 @@ function executeSpawn(
   }
 
   actorScope.defer(() => {
+    actorRef._parent = actorScope.self;
+    actorRef.system = actorScope.system;
```

Comment:
tbh, this looks like a massive hack

Reply:
There is also no test in the codebase that would prove this is needed. I can see how it might be needed for something but:
- we should have a test for it
- this should be resolved at some other time and not in this `execute` handler for the `spawnChild` action